debian/rules

```diff
@@ -7,18 +7,18 @@ export CFLAGS := $(shell dpkg-buildflags --get CFLAGS) $(shell dpkg-buildflags -
 export CXXFLAGS := $(shell dpkg-buildflags --get CXXFLAGS) $(shell dpkg-buildflags --get CPPFLAGS)
 export LDFLAGS := $(shell dpkg-buildflags --get LDFLAGS) -Wl,--as-needed
 export HOME := $(shell realpath .)
+export QT_SELECT=qt5
+
 DEB_HOST_MULTI_ARCH ?= $(shell dpkg-architecture -qDEB_BUILD_MULTIARCH)
 DEB_HOST_ARCH ?= $(shell dpkg-architecture -qDEB_HOST_ARCH)
-DEB_HOST_SERIES ?= $(shell lsb_release -c -s)
 
 ifneq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
 	# Cross compiling
 	CONFIGURE_OPTS += "CONFIG+=no_docs" "CONFIG+=cross_compile"
 endif
 
 # Skip tests on the archs they are known to be flaky  with current configuration
-testskip_architectures := armhf arm64 powerpc ppc64el s390x
-testskip_series := 
+testskip_architectures := armhf powerpc ppc64el s390x
```

Comment:
Tests still fail a lot on armhf then?

Reply:
Unfortunately, yes. It seems even though things are much less flaky on amd64 and arm64, they are still pretty flaky on armhf.